Packages/vcs/Lib/vtk_ui/textbox.py

```diff
@@ -47,7 +66,11 @@ def add_character(self, character):
         row = rows[self.row]
 
         if self.column >= len(row):
-            row += character
+            if self.blank:
```

Comment:
This was also addressed recently in VTK's master. Should be better once we bump. See Kitware/VTK@4d07c85e74a3458422faf900e56987983b943fb0